tests/integration/targets/ec2_vpc_net/tasks/main.yml

```diff
@@ -392,6 +392,7 @@
         state: present
         cidr_block: "{{ vpc_cidr }}"
         name: "{{ resource_prefix }}"
+        ipv6_cidr: True
```

Comment:
Note: when the `ipv6_cidr` wasn't set, that had the side effect of disabling the IPv6 association, because the default value was set to `false`.